spec/api/v1/external_users/claims/integration/litigator_claim_creation_spec.rb

```diff
@@ -389,6 +389,7 @@ def results(claim)
         post endpoint(:expenses), expense_params.merge(claim_id: claim.uuid, expense_type_id: expense_hotel.id), format: :json
         expect(last_response.status).to eql 201
         expect(claim.expenses.size).to eql 2
+        expect(claim.actual_trial_length).to eql 11
```

Comment:
This is comparing with `eql` (as is done throughout the rest of the tests in this file). This ignores the type, so `11.0` would be accepted. Is this what we want or should we be ensuring that the correct types are returned? Is it possible, for instance, that a client application could fail if a particular value starts being returned as a float?